### PR TITLE
[Linux] Add `game` category to instance shortcuts

### LINKED ISSUE #1443: Add "Game" to categories for created Linux desktop entries
### Role

I use the Prism Launcher to play Minecraft.

### Suggestion

I want the desktop entries that are created using the "Create Shortcut" button in Prism Launcher to add the `Categories=Game;` property to the created desktop entry.

### Benefit

On desktop systems that have application launcher menus, it will automatically sort it into a game's category (i.e. KDE plasma)

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

_No response_

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Trial97 — APPROVED** [review]

**Comment by ghost** [comment]
Tiny change, just needs one more review :P

File: launcher/FileSystem.cpp [hunk 1]
```diff
@@ -872,6 +872,8 @@ bool createShortcut(QString destination, QString target, QStringList args, QStri
            << "\n";
     stream << "Type=Application"
            << "\n";
+    stream << "Categories=Game"
```
Comment (getchoo) [inline launcher/FileSystem.cpp]: we should probably use the same categories as our regular [desktop file](https://github.com/PrismLauncher/PrismLauncher/blob/36d2109a1596df5ec20cb2b1828fc25846e7d926/program_info/org.prismlauncher.PrismLauncher.desktop.in#L10)

**Review by getchoo — CHANGES_REQUESTED** [review]

File: launcher/FileSystem.cpp (hunk 1 above)
Comment (ghost) [reply launcher/FileSystem.cpp]: Changes applied.

**Comment by Trial97** [comment]
please do not forget to sign your commits:https://github.com/PrismLauncher/PrismLauncher/pull/1704/checks?check_run_id=18443870695

**Comment by ghost** [comment]
Will that work?

**Comment by Trial97** [comment]
yes it worked thanks!

**Review by getchoo — APPROVED** [review]

**Pull request merged**